public/app/features/query/components/QueriesTab.tsx

```diff
@@ -273,22 +257,21 @@ export class QueriesTab extends PureComponent<Props, State> {
   };
 
   renderQueries() {
-    const { panel, dashboard } = this.props;
+    const { onQueriesChange, queries, onRunQueries } = this.props;
     const { dataSourceItem, data } = this.state;
 
-    if (isSharedDashboardQuery(dataSourceItem.name)) {
-      return <DashboardQueryEditor panel={panel} panelData={data} onChange={query => this.onUpdateQueries([query])} />;
-    }
+    // if (isSharedDashboardQuery(dataSourceItem.name)) {
+    //   return <DashboardQueryEditor panel={panel} panelData={data} onChange={query => this.onUpdateQueries([query])} />;
+    // }
```

Comment:
```suggestion
```

Reply:
ah, forgotten to fix this part. will add to the todo